lib/state.ml

```diff
@@ -5,5 +5,5 @@ type game_state = {
   board : board;
   players : player list;
   current_player_index : int;
+  words : string list;
```

Comment:
i think that adding the current list of words to the state object is not a robust solution b/c it duplicates information (now, the words that currently exist are stored in both the board and in this words list, and if they ever get out of sync then we will have a ton of bugs). instead, consider making a function that calculates the point increase given a `board` and an `entry_state` (see #36 and #37)

Reply:
are the words stored in board? I thought board only kept track of the cursor position and individual char tiles. I know that the words are displayed on the board, but there isn't a specific field that actually utilizes the `get_words` function to extract them for this type. Also, since I update this field upon use of the Enter key directly using the current board, how would they get out of sync? Using `board` and `entry_state` would require repeating point calculation for the same words (ie; even if the user only inputs one word the board would recalculate all word points twice), so I wonder if this would make the game slower.